makefile

```diff
@@ -0,0 +1,46 @@
+CC=g++
+CFLAGS=-std=c++17
+
+INCLUDECADMIUM=-I lib/cadmium/include
+
+#CREATE BIN AND BUILD FOLDERS TO SAVE THE COMPILED FILES DURING RUNTIME
+bin_folder := $(shell mkdir -p bin)
+build_folder := $(shell mkdir -p build/src build/test/subnet build/test/sender build/test/receiver)
+
+#TARGET TO RUN ONLY BIT SIMULATOR
+simulator: main.o ../data_structures/message.o
+	$(CC) -g -o bin/ABP build/src/main.o build/src/message.o
+clean_simulator:
+	rm -f bin/ABP.exe build/src/*.o *~
+	-for d in src/data_structures; do (cd $$d; rm -f *.o *~ ); done
```

Comment:
you can remove this line because we aren't generating .o files in data_structures folder anymore.